routes/users.router.js

```diff
@@ -40,13 +40,221 @@ const router = express.Router();
  *            - lastName
  *            - email
  *            - password
- * 
+ *
  */
 
-router.post("/", validateRequestSchema(createUserSchema), createUsers);
+/**
+ * @swagger
+ * /users:
+ *   get:
+ *     summary: Find all users
+ *     tags: [User]
+ *     description: Find all users.
+ *     parameters:
+ *        - name: page
+ *          in: query
+ *          description: Filter by page.
+ *          schema:
+ *             type: integer
+ *             format: int64
+ *             example: 1
+ *     responses:
+ *       '200':
+ *            description: Ok - successful operation
+ *            content:
+ *               application/json:
+ *                 schema:
+ *                   type: array
+ *                   items:
+ *                     $ref: '#/components/schemas/User'
+ *                   example:
+ *                     status: true
+ *                     code: 200
+ *                     message: Users obtained successfully
+ *                     body:
+ *                        firstName: Mandy
+ *                        lastName: Pyford
+ *                        email: mpyford0@xinhuanet.com
+ *                        createdAt: 2022-11-10T21:45:49.000Z       
+ *   post:
+ *     summary: Create user
+ *     tags: [User]
+ *     description: This can only be done by the logged in user.
```

Comment:
Correct this please. This endpoint is for register an user, you don't have to be logged to do this.
You can put  Register an user